Tools/deafrica_tools/datahandling.py

```diff
@@ -393,7 +401,14 @@ def load_ard(
         # Obtain list of datasets for product
         if verbose:
             print(f"    {product}")
-        datasets = dc.find_datasets(product=product, **query)
+        
+        if product_type == "ls":
+            #handle LS seperately to S2/S1 due to collection_category
+            datasets = dc.find_datasets(product=product,
+                                    collection_category=collection_category,
```

Comment:
If collection_category is not also provided in **kwargs, this might break?
What if we check the kwargs dictionary and set it to T1 if it's not set?